src/Razor/test/testapps/ClassLibrary2/wwwroot/js/project-direct-dep.js

```diff
@@ -0,0 +1,3 @@
+(function () {
+  document.getElementById('project-direct-dep').innerHTML = 'project-direct-dep';
+})()
```

Comment:
Why is the cache file ⬇️  getting checked in?

Reply:
What do you mean?

Reply:
Nevermind. The previous PR had a .cache file below being checked in.